static/docs/get-started/initialize.md

```diff
@@ -1,12 +1,11 @@
 # Initialize
 
-In order to start using DVC, you need first to initialize it in your
-<abbr>workspace</abbr>. DVC doesn't require Git and can work without any source
-control management system, but for the best experience we recommend using DVC on
-top of Git repositories.
+There are a few recommended ways to install DVC: OS-specific package/installer,
+`pip`, `conda`, and Homebrew. See the **[installation](/doc/install)** page for
+all the options and details.
 
-If you don't have a directory for this <abbr>project</abbr> already, create it
-now with these commands:
+In order to start using DVC, you need first to initialize it in your
+<abbr>workspace</abbr>:
 
 ```dvc
 $ mkdir example-get-started
```

Comment:
I agree this paragraph could be improved but this new wording seems also inexact. I'll revise it.

Reply:
Improved in https://github.com/iterative/dvc.org/pull/734#pullrequestreview-306996048.